letsencrypt-win-simple/Services/PluginService.cs

```diff
@@ -102,13 +103,26 @@ public PluginService()
 
         private List<Type> GetResolvable<T>(bool allowNull = false)
         {
-            var ret = Assembly.GetExecutingAssembly()
-                        .GetTypes()
-                        .Where(type => typeof(T) != type && typeof(T).IsAssignableFrom(type) && !type.IsAbstract);
-            if (!allowNull)
+            var baseDir = AppDomain.CurrentDomain.BaseDirectory;
+            string[] extensions = { ".dll" };
+            var allFiles = Directory.EnumerateFileSystemEntries(baseDir, "*.*")
+                .Where(x => extensions.Any(ext => ext == Path.GetExtension(x)));
+            var assemblies = new List<Assembly> { Assembly.GetExecutingAssembly() };
+            assemblies.AddRange(allFiles.Select(AssemblyName.GetAssemblyName).Select(Assembly.Load));

```

Comment:
Maybe add some try-catch with a Log.Warn if it fails to load? (Not sure if the logger is available at this point)

Reply:
Yep, agreed, couldn't figure out how to get the logger, but I'll look into it again.